docs/post-merge

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+import sys
+import os.path
+
+file_path = "mms5-layer1-service/src/main/resources/application.conf"
+if os.path.exists(file_path):
+    # open both files
+    app_example = "mms5-layer1-service/src/main/resources/application.conf.example"
+    with open(file_path,'r') as firstfile, open(file_path,'w') as secondfile:
```

Comment:
I think this is meant to be opening the application.conf.example file?

Reply:
Yes, I'll edit the comment to reflect that.

Reply:
`application.conf.example` should be copied to `application.conf`, not the other way around. Also, it should only write to the file if it does not exist. I'd suggest using `'x'` file flag in the call to `open` with a try catch.

Reply:
It looks like the application.conf is a hidden file and actually does exist within that directory. The issue was that it was empty and so that would produce the original error. Because of this, the 'w' flag would be more appropriate as the 'x' flag will cause a "FileAlreadyExists" error.  I have added the changes.